test_runner/src/test/kotlin/ftl/args/AndroidArgsTest.kt

```diff
@@ -2193,7 +2193,7 @@ AndroidArgs
             - model: blueline
               version: 28
               locale: en
-              orientation: poortrait
+              orientation: portrait_
```

Comment:
I prefer to use `_` to indicate "this isn't the real word" while still making it easy for spell checking tools to find typos. But, I'm happy to drop these changes...